{{cookiecutter.repo_name}}/config/wsgi.py

```diff
@@ -36,6 +39,9 @@
 # See: https://whitenoise.readthedocs.org/
 application = DjangoWhiteNoise(application)
 {%- endif %}
+{% if cookiecutter.use_sentry == "y" -%}
+application = Sentry(application)
```

Comment:
is this really necessary? if use_whitenoise is enable, this overwrites the previous `application`
some documentation about it?